pagopa/io-dev-api-server
# [IOCOM-1886] Autogenerated FIMS history specs

⚠️ This PR depends on #431 ⚠️

## Short description
This PR adds the autogeneration of FIMS history specs.

## List of changes proposed in this pull request
- Automatic generation of FIMS history specs
- Proper coding of both access list and export request for FIMS

## How to test
Check that both endpoints
- `get` `/api/v1/fims/accesses`
- `post` `/api/v1/fims/export-requests`

work as expected
